### PR TITLE
[sui-ssr] Add Device to context

### PR DESCRIPTION
**Pull request by carlosvillu**

### CODE REVIEW

File: packages/sui-ssr/build-device.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const buildDeviceFrom = ({request = {}, window = {}}) => {
+  return {
+    // https://tc39.github.io/proposal-optional-chaining/
+    'user-agent':
```
Comment (miduga) [inline packages/sui-ssr/build-device.js]: Why not returning it camelCase?

Comment (carlosvillu) [reply packages/sui-ssr/build-device.js]: No reason, just because is the header´s name

Comment (carlesnunez) [reply packages/sui-ssr/build-device.js]: If is not a huge change or painful I'll change it by camelCase in order to be more compliant with the var names standards

File: packages/sui-ssr/build-device.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const buildDeviceFrom = ({request = {}, window = {}}) => {
```
Comment (jcger) [inline packages/sui-ssr/build-device.js]: IMO something like `buildDeviceInfo`, `getDeviceData` or even just `buildDevice` would be more descriptive

Comment (carlosvillu) [reply packages/sui-ssr/build-device.js]: The idea is combine the name of the method with the param name, to get something like: `buildDeviceFrom({window})`

To achive a more semantic method call

**Pull request merged**